src/test/groovy/org/prebid/server/functional/model/config/PurposeEid.groovy

```diff
@@ -0,0 +1,14 @@
+package org.prebid.server.functional.model.config
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies
+import com.fasterxml.jackson.databind.annotation.JsonNaming
+import groovy.transform.ToString
+
+@ToString(includeNames = true, ignoreNulls = true)
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy)
```

Comment:
Are we sure that this should be snake_case? It's an account config, right? We were trying to have kebab-case for account config properties.

Reply:
it's a requirement from [issue](https://github.com/prebid/prebid-server/issues/2904#:~:text=account%2Dlevel%20config-,privacy.gdpr.purposes.p4.eid.require_consent%3A%20true/false%20%20//%20if%20true%2C%20eids%20require%20P4%20legal%20basis%20unless%20excepted%20below,-privacy.gdpr.purposes)

```
privacy.gdpr.purposes.p4.eid.require_consent: true/false  // if true, eids require P4 legal basis unless excepted below
privacy.gdpr.purposes.p4.eid.exceptions: ["pubcid.org"]   // doesn't do anything when eids-require-consent: false.
```